docker-centos/config.json.template

```diff
@@ -362,6 +362,11 @@
 	    ]
 	}
     ],
+    "renameFiles" : {
```

Comment:
This is a system container feature that should go in a separate file `manifest.json'.
For example we have this `manifest.json` file for the Flannel container: https://github.com/projectatomic/atomic-system-containers/blob/master/flannel/manifest.json